front/src/js/activity_bar.js

```diff
@@ -1,11 +1,30 @@
 const activityBar = {
     name: "activity-bar",
     props: ["currentRoom", "isWriting"],
+    methods:{
+        openPopup: function() { 
```

Comment:
it seem that with the vue v-show you don't need that function so delete it

Reply:
yes i  have forgot to remove

Reply:
happen a lot to me too :joy:

Reply:
done here d4b541e